packages/google_atlas/example/main.dart

```diff
@@ -58,6 +58,18 @@ class AtlasSample extends StatelessWidget {
             )
           ],
         ),
+        circles: Set<Circle>.from(
```

Comment:
Minor but this can be written like:
```dart
circles: <Circle>{
  Circle(
    id: 'circle-1',
    center: LatLng(
      latitude: 38.7439498,
      longitude: -9.1490721,
    ),
    radiusInMeters: 10.0,
  ),
},

Reply:
There are more instance of this and I was going to change them. But on the google_atlas package I get a warning to update the SDK constraints, since set literals are only support after 2.2.
Can I update the SDK constraints? It's require to also update the same constraint on the atlas package?

Reply:
I'm fine with that. The latest stable version of Flutter supports Dart 2.5, so we should be fine upgrading that constraint